src/test/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/PaymentAccountValidatorTest.java

```diff
@@ -67,13 +65,4 @@ public void testValidatePaymentAccounts() {
         paymentAccounts.add(pba);
         paymentAccountValidator.validatePaymentAccounts(paymentAccounts, "");
     }
-
-
-    @Test
-    public void privateConstructorTest() throws Exception {
```

Comment:
This was removed as `PaymentAccountValidator` does not have a private constructor